docs/sphinx_doc/en/source/tutorial/208-distribute.md

```diff
@@ -315,6 +322,65 @@ When running large-scale multi-agent applications, it's common to have multiple
         ok = client.delete_all_agent()
     ```
 
+#### Connecting to AgentScope Studio
+
+The agent server process can be connected to [AgentScope Studio](#209-gui-en) at startup, allowing the `to_dist` method in subsequent distributed applications to be assigned automatically by Studio without the need for any parameters.
```

Comment:
What if the server or the main process (where the user run the application) didn't connect to the studio ?

Reply:
Degenerate to Subprocess mode